src/cpp/lib/QtWidgets/QTableWidget/qtablewidget_wrap.cpp

```diff
@@ -34,6 +34,27 @@ Napi::Object QTableWidgetWrap::init(Napi::Env env, Napi::Object exports) {
        InstanceMethod("insertRow", &QTableWidgetWrap::insertRow),
        InstanceMethod("removeRow", &QTableWidgetWrap::removeRow),
        InstanceMethod("scrollToItem", &QTableWidgetWrap::scrollToItem),
+       InstanceMethod("cellWidget", &QTableWidgetWrap::cellWidget),
+       InstanceMethod("column", &QTableWidgetWrap::column),
+       InstanceMethod("row", &QTableWidgetWrap::row),
+       InstanceMethod("currentColumn", &QTableWidgetWrap::currentColumn),
+       InstanceMethod("currentItem", &QTableWidgetWrap::currentItem),
+       InstanceMethod("currentRow", &QTableWidgetWrap::currentRow),
+       InstanceMethod("findItems", &QTableWidgetWrap::findItems),
+       InstanceMethod("item", &QTableWidgetWrap::item),
+       InstanceMethod("itemAt", &QTableWidgetWrap::itemAt),
+       InstanceMethod("removeCellWidget", &QTableWidgetWrap::removeCellWidget),
+       InstanceMethod("setCurrentCell", &QTableWidgetWrap::setCurrentCell),
+       InstanceMethod("setCurrentItem", &QTableWidgetWrap::setCurrentItem),
+       InstanceMethod("sortItems", &QTableWidgetWrap::sortItems),
+       InstanceMethod("takeItem", &QTableWidgetWrap::takeItem),
+       InstanceMethod("visualColumn", &QTableWidgetWrap::visualColumn),
+       InstanceMethod("visualItemRect", &QTableWidgetWrap::visualItemRect),
+       InstanceMethod("visualRow", &QTableWidgetWrap::visualRow),
+       InstanceMethod("columnCount", &QTableWidgetWrap::columnCount),
+       InstanceMethod("rowCount", &QTableWidgetWrap::rowCount),
+       InstanceMethod("setColumnCount", &QTableWidgetWrap::setColumnCount),
+       InstanceMethod("setRowCount", &QTableWidgetWrap::setRowCount),
```

Comment:
Did we have a whole bunch of code in there which just was never hooked up to the nodejs wrapper?

Reply:
Yes, I added the 4 actual missing functions that everything got somehow removed, and then re-wired the 15 or so missing `InstanceMethod` that should have been tied to code that still existed in both the C/H & TS files -- but somehow lost the `InstanceMethod`s.   So that code was failing from the TS/JS side since they weren't actually wired up...

Reply:
I should probably also mention the existing code still works and was originally from one of my prior PR's.   It was hooked up before the refactoring, as I've been using several of those functions with QT 5.  I finally upgraded the library to the current version and found all the broken issues.    

My program seems to now be working again with the changes I've made into this PR.